spec/models/child_theme_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe ChildTheme do
+  it "doesn't allow a user selectable theme to be a child" do
+    parent = Theme.create!(name: "parent", user_id: -1)
+    selectable_theme = Theme.create!(name: "selectable", user_id: -1, user_selectable: true)
+
+    relation = ChildTheme.new(parent_theme: parent, child_theme: selectable_theme)
```

Comment:
I'm not sure if `relation` is the right variable name here as the object is an ActiveRecord object. I would just name it `child_theme`